bin/compile.js

```diff
@@ -1,34 +0,0 @@
-const fs = require('fs-extra');
-const debug = require('debug')('app:bin:compile');
-const webpackCompiler = require('../build/webpack-compiler');
-const webpackConfig = require('../build/webpack.config');
-const config = require('../config');
-
-const paths = config.utils_paths;
-
-const compile = () => {
-  debug('Starting compiler.');
-  return Promise.resolve()
-    .then(() => webpackCompiler(webpackConfig))
-    .then(stats => {
-      if (stats.warnings.length && config.compiler_fail_on_warning) {
-        throw new Error('Config set to fail on warning, exiting with status code "1".');
-      }
-      try {
-        fs.accessSync(paths.client('static'), fs.F_OK);
-        debug('Copying static assets to dist folder.');
-        fs.copySync(paths.client('static'), paths.dist());
-      } catch (e) {
-        debug('No static assets folder found, continuing...');
-      }
-    })
-    .then(() => {
-      debug('Compilation completed successfully.');
-    })
-    .catch((err) => {
-      debug('Compiler encountered an error.', err);
-      process.exit(1);
-    });
-};
-
-compile();
```

Comment:
How are we going to build and publish our actual components now? 🤔

Reply:
If I'm not mistaken, the components get build using the `npm run build` script which doesn't use those bin scripts but gulp.

Reply:
@Sambego I'm sorry, you are right. I'm confusing different projects and their setups. Can you also make sure you remove the (now unused) packages? Thanks! 👍 

 :ok: by @nickwaelkens on 11/10/2017 13:12:45

Reply:
Done!

Reply:
Cool, I will make a follow-up PR that cleans out even more stuff (out of scope for now)